Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -2475,7 +2475,11 @@ def normalize_scan_data(scan_data: dict) -> dict:
         include_none=True,
     )
 
-    result["TotalTime"] = readable_duration_time(scan_data["duration"])
+    if "duration" in scan_data:
+        result["TotalTime"] = readable_duration_time(scan_data["duration"])
+
+    else:
+        result["TotalTime"] = "No duration data was found."
```

Comment:
@bziser I've checked and debugged the old integration by checking out commit 6d2266fd29997611a64988f9362faa349e8ab4c5, and it seems like previously, if the `duration` key was missing, the value of `TotalTime` ended up being `0 minutes`.

I thought returning a message (since its value is a string anyway) saying it couldn't be found would be more accurate, but I guess it could be considered as breaking BC.

Please let me know what you think.
(see PR's description for more context about the bug and why I believe it happened)